### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by pbradham**

### CODE REVIEW

**Comment by redouane-dziri** [comment]
nearly there, delete auto folder, template.tex~, change the date in template.tex (date of lecture, not today's date) and regenerate pdf

**Comment by redouane-dziri** [comment]
delete auto folder, change title in template.tex
This was very straighforward in the readme..

File: lecture_4/pmb2164/pmb2164.tex [hunk 1]
```diff
@@ -0,0 +1,53 @@
+
+% use a sans serif font
+\renewcommand{\familydefault}{\sfdefault}
```
Comment (jhofman) [inline lecture_4/pmb2164/pmb2164.tex]: Remove this renewcommand line, it's defined in the template.tex file

File: lecture_4/pmb2164/pmb2164.tex [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+\begin{document}
```
Comment (jhofman) [inline lecture_4/pmb2164/pmb2164.tex]: remove \begin{document} and \maketitle as well. they're included in template.tex too.

File: lecture_4/pmb2164/pmb2164.tex [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+\subsection{R Studio: Working with ggplot}
+During the last part of lecture, Professor Hofman quickly went over how to setup ggplot, and different types of plots and examples of such. First make sure you are running the Tidyverse, lubridate and scales libraries to start. Note that the ggplot histogram doesn't require a y, it associates the y as the count of the x function. Please, when you're reading this, open Professor Hofman's Github hub for visuals. We talked about how make bar charts, a normal histogram, a scatter plot histogram, line plots, and many more. While going through the examples keep in mind what is trying to be shown, and how each graph has its perks and drawbacks. We also go into to how to show question 3 on the homework, hint hint hint. And also how to label the charts, how to add color on the charts, and how to add error bars and confidence intervals. Please, Please go through the Github page, please it will be very beneficial
+
+
+
+El Fin
+
+   
+\end{document}
```
Comment (jhofman) [inline lecture_4/pmb2164/pmb2164.tex]: you don't need this either.

**Pull request merged**